docker/docker-compose.middleware.yaml

```diff
@@ -65,6 +65,7 @@ services:
       ENABLE_NETWORK: 'true'
       HTTP_PROXY: 'http://ssrf_proxy:3128'
       HTTPS_PROXY: 'http://ssrf_proxy:3128'
+      SANDBOX_PORT: 8194
```

Comment:
btw, `SANDBOX_PORT` is not a best env name, it does not introduce any useful concept to this, neither for usage or hints.